src/mainClass.js

```diff
@@ -2,17 +2,24 @@ import { getData, getAllCategories } from "./source";
 
 export class AllItems {
 
-  constructor(searchCategory, searchLinks, displayLinksForChosenCategory) {
+  constructor(renderCategories, renderLinks) {
```

Comment:
renderCategories, renderLinks I not like naming, name it same as prop: onReceivedCategories and onReceivedLinks